src/lib/native.ts

```diff
@@ -422,10 +422,10 @@ export function parseProfilesIni(content: string, basePath: string) {
     for (const profileName of Object.keys(result)) {
         const profile = result[profileName]
         // profile.IsRelative can be 0, 1 or undefined
-        if (profile.IsRelative === 1) {
+        if (profile.IsRelative === "1") {
```

Comment:
`tslint --fix` broke some stuff I guess?

Reply:
Nope, that's me, turning `==` into `===` has to be done manually. I didn't pay enough attention to profile.IsRelative's type before adding a third equals sign.